src/main/java/com/netchill/services/streaming/StreamService.java

```diff
@@ -64,9 +64,32 @@ public long[] getRangePart(String range, long videoLength) {
     public byte[] getVideoPart(Long id, long videoStart, long videoEnd) throws IOException {
         InputStream videoPath = StreamService.class.getResourceAsStream("/videos/" + this.movieDao.getMovieUrl(id));
 
-        //Je n'ai pas trouvé de fonction qui renvoie le bout de vidéo d'un début x à une fin y, donc je skip la vidéo jusqu'au début du range.
-        videoPath.skipNBytes(videoStart);
-        return videoPath.readNBytes((int) videoEnd);
+        long remainingBytesToSkip = videoStart;
+        // Utilisatioin d'une boucle car ce n'est pas sur que le saut se fait en une fois, il faut donc gérer cela
+        while (remainingBytesToSkip > 0) {
+            System.out.println("SKIPPED");
+            long bytesSkipped = videoPath.skip(remainingBytesToSkip);
+            if (bytesSkipped <= 0) {
+                throw new IOException("Impossible de sauter jusqu'au début de la plage spécifiée.");
+            }
+            remainingBytesToSkip -= bytesSkipped;
+        }
+
+        int bufferSize = CHUNK_SIZE; // Taille du tampon de lecture
```

Comment:
Why don't you use the contant `CHUNK_SIZE` directly instead of storing it in a new variable ?